core/domain/user_services.py

```diff
@@ -42,7 +42,6 @@
 
-
```

Comment:
There should be two blank lines before a class declaration.

Reply:
Done